awx/main/models/inventory.py

```diff
@@ -803,6 +803,14 @@ def _get_related_jobs(self):
         return UnifiedJob.objects.non_polymorphic().filter(Q(job__inventory=self.inventory) | Q(inventoryupdate__inventory_source__groups=self))
 
 
+class HostMetrics(models.Model):
```

Comment:
This is a little nitpicky, but for models like this `Host`, `Inventory`, etc... we generally keep this name singular i.e.,

`class HostMetric` since *each one* represents _one_ metric.

Reply:
I have changed it to ```HostMetric```